opossum-tool/OpossumUI
# feat: use Opossum colors in theme palette

### Summary of changes

- to properly leverage the "color" prop on many MUI components, we set the theme palette to the values used by OpossumUI
- this also means that MUI correctly and automatically calculates the required states for `light`, `dark`, `contrastText` instead of us having to supply them: https://mui.com/material-ui/customization/palette/#customization

### Context and reason for change

Following MUI best practices and improving button states. Also required as part of #2332.

### How can the changes be tested

Check buttons throughout OpossumUI that they still look as before but have improved hover/active states (calculated by MUI now).
